Connectors/src/PostgreSqlEFCore/README.md

```diff
@@ -0,0 +1,65 @@
+﻿# PostgreSQL Connector Sample App - Entity Framework Core
+
+ASP.NET Core sample app illustrating how to use Entity Framework Core together with [Steeltoe PostgreSQL Connector](https://docs.steeltoe.io/api/v3/connectors/postgresql.html#add-dbcontext) for connecting to a PostgreSQL service on CloudFoundry.
+There is also an additional sample that illustrates how to use a `NpgsqlConnection` to issue commands to the bound database.
+
+## General prerequisites
+
+1. Installed .NET Core SDK
+
+## Running locally
+
+1. Started PostgreSQL [docker container](https://github.com/SteeltoeOSS/Samples/blob/main/CommonTasks.md)
+
+## Running on CloudFoundry
+
+1. Installed CloudFoundry (optionally with Windows support)
+1. Installed [VMware Tanzu Cloud Service Broker](https://docs.vmware.com/en/Cloud-Service-Broker-for-VMware-Tanzu/index.html)
+
+## Create PostgreSQL Service Instance on CloudFoundry
+
+You must first create an instance of the PostgreSQL service in an org/space.
+
+1. `cf target -o your-org -s your-space`
+1. `cf create-service csb-azure-postgresql small myPostgreSqlService` or `cf create-service csb-google-postgres default myPostgreSqlService`
```

Comment:
Did you run into any issues with Postgres provisioned by the CSB for GCP? I have a distant memory of issues regarding required use of client certificates

Reply:
I don't think it is an immediate priority to have it working, we can tackle that separately. The Cloud SQL Auth proxy looks interesting, but I suspect it won't be something available for use with the cloud service broker.